chef/automate
# Runs request returns associated projects

### :nut_and_bolt: Description: What code changed, and why?
Adding the associated projects to a requested API call for a client run. The API URL is "api/v0/cfgmgmt/nodes/<node_id>/runs/<run_id>". Since a run does not store its associated projects, the run's node must be retrieved from Elasticsearch also. 

### :chains: Related Resources
#1929

### :+1: Definition of Done
When requesting a run from the "api/v0/cfgmgmt/nodes/<node_id>/runs/<run_id>" endpoint the run's associated projects are returned. 

### :athletic_shoe: How to Build and Test the Change
1. `build components/config-mgmt-service && build components/automate-gateway && start_all_services`
1. Add some nodes with `chef_load_nodes 10`
1. Create a project with ingestion rules that match at least one of the nodes. 
1. Update the projects and go to the Client Runs page https://a2-dev.test/infrastructure/client-runs
1. Open the browser inspect window and select the "Network" tab. 
1. Filter the request for "api/v0/cfgmgmt/nodes/"
1. Select the node in the node list that the project ingestion rule was created to match. 
1. Select the new row in the inspect window for "api/v0/cfgmgmt/nodes/<node_id>/runs/<run_id>"
1. Ensure in the "Response" tab that the correct projects are returned like in the screenshot below. 


### :white_check_mark: Checklist


### :camera: Screenshots, if applicable
![Screen Shot 2019-10-22 at 4 05 59 PM](https://user-images.githubusercontent.com/1679247/67342037-13999b00-f4e6-11e9-94b6-08d58c466993.png)
